config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'dishes/show'
```

Comment:
Should this really be the show page? We normally use that for individual dishes. This is the kind of info we would ordinarily put in an index for dishes. So url.com/dishes takes you to a list of all dishes and when you want to view an individual dish, it's url.com/dishes/1 - and that's what the `show` method is for.